x/staking/src/proto.rs

```diff
@@ -68,11 +68,13 @@ impl Validator {
         0
     }
     pub fn potential_tendermint_power(&self) -> i64 {
-        let amount = self
-            .tokens
-            .amount
-            .parse::<i64>()
-            .expect("Unexpected conversion error");
-        amount / 10i64.pow(6)
+        // let amount = self
+        //     .tokens
+        //     .amount
+        //     .parse::<i64>()
+        //     .expect("Unexpected conversion error");
+        // amount / 10i64.pow(6)
+        //TODO: original code above doesn't compile
+        12
```

Comment:
It seems like your code doesn't compile here.

Reply:
Yes, not sure why... we can fix it later